config/locales/en.yml

```diff
@@ -1981,6 +1972,15 @@ en:
           - You didn't receive the full amount of stimulus you were owed
           right_for_you: 'This is the right tool for you if:'
           title: Welcome!
+        prior_tax_year_agi:
+          help_text_html: |
+            Because you stated that you filed a full tax return in 2019, we’ll need to verify what your Adjusted Gross Income (AGI) was in 2019.</br>
```

Comment:
im going to change all of these in another PR where I change all of the references -- it ended up muddying the PR and taking a long time, especially since there were enough changes in this PR.